gQuant/docker/build.sh

```diff
@@ -61,54 +71,96 @@ case $CUDA_VERSION in
         ;;
 esac
 
+mkdir -p ${BUILDDIR}
+cp -r ${GREENFLOWDIR} ${BUILDDIR}
+rsync -av --progress ${GREENFLOWLABDIR} ${BUILDDIR} --exclude node_modules 
+# cp ${TOPDIR}/gQuant/greenflowrc ${BUILDDIR}
+cp ${TOPDIR}/gQuant/README.md ${BUILDDIR}
+
+# cp -r ${PLUGINSDIR} ${BUILDDIR}
+mkdir -p "${BUILDDIR}/plugins"
+rsync -av --progress "${PLUGINSDIR}/gquant_plugin" "${BUILDDIR}/plugins" \
+  --exclude data \
+  --exclude .cache \
+  --exclude many-small \
+  --exclude storage \
+  --exclude dask-worker-space \
+  --exclude __pycache__
+
+rsync -av --progress "${PLUGINSDIR}/dask_plugin" "${BUILDDIR}/plugins" \
+  --exclude data \
+  --exclude .cache \
+  --exclude many-small \
+  --exclude storage \
+  --exclude dask-worker-space \
+  --exclude __pycache__
+
+rsync -av --progress "${PLUGINSDIR}/simple_example" "${BUILDDIR}/plugins" \
+  --exclude data \
+  --exclude .cache \
+  --exclude many-small \
+  --exclude storage \
+  --exclude dask-worker-space \
+  --exclude __pycache__
+
+rsync -av --progress "${PLUGINSDIR}/nemo_plugin" "${BUILDDIR}/plugins" \
+  --exclude data \
+  --exclude .cache \
+  --exclude many-small \
+  --exclude storage \
+  --exclude dask-worker-space \
+  --exclude __pycache__
 
 read -p "Enable dev model [y/n]:" DEV_MODE
 case $DEV_MODE in
     y)
-	echo "Dev mode"
+    echo "Dev mode"
     read -r -d '' INSTALL_GREENFLOW<< EOM
 ## copy greenflowlab extension
-ADD --chown=$USERID:$USERGID ./greenflow /home/quant/greenflow
+ADD --chown=$USERID:$USERGID ./build /home/quant/greenflow
 WORKDIR /home/quant/greenflow
 EOM
     MODE_STR="dev"
-	;;
+    ;;
     *)
-	echo "Production mode"
+    echo "Production mode"
     read -r -d '' INSTALL_GREENFLOW<< EOM
+
+WORKDIR /home/quant
+
+ADD --chown=$USERID:$USERGID ./build/README.md /home/quant/README.md
+
+## install greenflow
+ADD --chown=$USERID:$USERGID ./build/greenflow /home/quant/greenflow
+RUN cd /home/quant/greenflow && pip install .
+
 ## install greenflowlab extension
-ADD --chown=$USERID:$USERGID ./greenflow /home/quant/greenflow
-RUN pip install .
-WORKDIR /home/quant/greenflow/greenflowlab
-RUN pip install .
+ADD --chown=$USERID:$USERGID ./build/greenflowlab /home/quant/greenflowlab
+RUN cd /home/quant/greenflowlab && pip install .
+
 RUN jupyter lab build
-WORKDIR /home/quant/greenflow
-ENTRYPOINT MODULEPATH=\$HOME/greenflow/modules jupyter-lab --allow-root --ip=0.0.0.0 --no-browser --NotebookApp.token=''
+
+## install greenflow plugins
+ADD --chown=$USERID:$USERGID ./build/plugins /home/quant/plugins
+RUN cd /home/quant/plugins/gquant_plugin && pip install .
+RUN cd /home/quant/plugins/dask_plugin && pip install .
+
+WORKDIR /home/quant
+ENTRYPOINT MODULEPATH=\$HOME/plugins/gquant_plugin/modules jupyter-lab \
```

Comment:
If we start the jupyter-lab at the /home/quant directory, the notebooks won't properly.
Change the path to 
```python
WORKDIR /home/quant/plugins/gquant_plugin
ENTRYPOINT MODULEPATH=\$HOME/plugins/gquant_plugin/modules jupyter-lab \
  --allow-root --ip=0.0.0.0 --no-browser --NotebookApp.token='' \
  --ContentsManager.allow_hidden=True
```

Reply:
Fixed in commit 563ddde.